### PR TITLE
cpufreq@mtwebster: Make panel frequency formatting width stable (#.##)

### PR DESCRIPTION
**Pull request by yrtimiD**
Fixes cpufreq@mtwebster: make stable and localized frequency format to avoid layout shifts #6064

@mtwebster , will appreciate your review

### CODE REVIEW

File: cpufreq@mtwebster/files/cpufreq@mtwebster/applet.js [hunk 1]
```diff
@@ -92,13 +94,20 @@ function rd_nums_frm_file(file) {
...
+        num = Math.round(num / 10) / 100;
+        units = 'MHz';
+    } else if (num < 1000000000) {
+        num = Math.round(num / 10000) / 100;
+        units = 'GHz';
+    } else {
+        num = Math.round(num / 10000000) / 100;
+        units = 'THz';
+    }
+    return formatNum(num) + ' ' + units;
```
Comment (rcalixte) [inline cpufreq@mtwebster/files/cpufreq@mtwebster/applet.js:110]: There is a built-in function for this instead.
```suggestion
    return num.toFixed(2) + ' ' + units;
```

File: cpufreq@mtwebster/files/cpufreq@mtwebster/applet.js [hunk 2]
```diff
@@ -66,6 +66,8 @@ const cpu_dir = Gio.file_new_for_path(cpu_path);
 let height = 22;
 const DEC_WHITE = 16777215;
 
+const formatNum = new Intl.NumberFormat(undefined, { maximumFractionDigits: 2, minimumFractionDigits: 2 }).format;
```
Comment (rcalixte) [inline cpufreq@mtwebster/files/cpufreq@mtwebster/applet.js:69]:
```suggestion

```

Comment (yrtimiD) [reply cpufreq@mtwebster/files/cpufreq@mtwebster/applet.js:69]: Isn't better to format numbers using locale aware framework? For example, some locales use comma for decimal separators.
(Just checked, Cinnamon applets runs with the locale taken from the system)

Comment (yrtimiD) [reply cpufreq@mtwebster/files/cpufreq@mtwebster/applet.js:69]: actually, I'd also change all other number presentations to use this formatter (will keep it for next PR)

Comment (yrtimiD) [reply cpufreq@mtwebster/files/cpufreq@mtwebster/applet.js]: sorry for the noise, just an indentation reformat

**Review by rcalixte — APPROVED** [review]

**Comment by rcalixte** [comment]
@yrtimiD This applet is technically unmaintained. Are you interested in taking over as the maintainer?

**Comment by yrtimiD** [comment]
> @yrtimiD This applet is technically unmaintained. Are you interested in taking over as the maintainer?

I'm not much familiar with cinnamon applets development, but can try to help with some open bugs and future reviews.

**Pull request merged**